src/efuns.c

```diff
@@ -8505,6 +8513,46 @@ f_configure_driver (svalue_t *sp)
             swap_compact_mode = (sp->u.number != 0);
             break;
 
+        case DC_SWAP_TIME:
+            if (sp->type != T_NUMBER)
+                efun_arg_error(2, T_NUMBER, sp->type, sp);
+            if (sp->u.number < 0)
+            {
+                errorf("Time to swap programs must be >= 0!\n");
+            }
+            time_to_swap = sp->u.number;
+            break;
+
+        case DC_SWAP_VAR_TIME:
+            if (sp->type != T_NUMBER)
+                efun_arg_error(2, T_NUMBER, sp->type, sp);
+            if (sp->u.number < 0)
+            {
+                errorf("Time to swap variables must be >= 0!\n");
+            }
+            time_to_swap_variables = sp->u.number;
+            break;
+
+        case DC_CLEANUP_TIME:
+            if (sp->type != T_NUMBER)
+                efun_arg_error(2, T_NUMBER, sp->type, sp);
+            if (sp->u.number < 0)
+            {
+                errorf("Time to call cleanup hook must be >= 0!\n");
+            }
+            time_to_cleanup = sp->u.number;
```

Comment:
Wenn ich das richtig sehe, ändert das nicht die Defines _RESET_TIME__ bzw. __CLEANUP_TIME__ in LPC (siehe init_lexer()). Einfachste Lösung: Statt einer Konstanten eine Funktion nutzen, die die aktuelle Variable abfragt.